src/ui/pages/RevealPrivateKey/usePrivateKey.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query';
+import { invariant } from 'src/shared/invariant';
+import { walletPort } from 'src/ui/shared/channels';
+
+async function getPrivateKey(address: string) {
+  return walletPort.request('getPrivateKey', { address });
```

Comment:
This can be an inline call inside usePrivateKey.
I think, we don't need to wrap port requests in general

Reply:
Post-refactoring cringe moments... thanks! ✅